qubes/storage/callback.py

```diff
@@ -509,17 +509,17 @@ def __init__(self, pool, impl):
         self._cb_impl = impl  #: Backend volume implementation instance.
 
     async def _assert_initialized(self, **kwargs):
-        await self._cb_pool._assert_initialized(
+        await self._cb_pool._assert_initialized(  # pylint: disable=protected-access
```

Comment:
shouldn't this be before the call then?

Reply:
I can try before but at this place it works too. Let me try.